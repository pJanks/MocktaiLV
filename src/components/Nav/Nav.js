import React from 'react';
import { Link } from 'react-router-dom';


const Nav = () => {
  return (
    <nav>
    <button><Link className='link' to='/'>Homepage</Link></button>
    <button><Link className='link' to='/OffTheWagon'>Off The Wagon</Link></button>
    <button><Link className='link' to='/Favorites'>Favorites</Link></button>
    </nav>
  )
}

export default Nav;
