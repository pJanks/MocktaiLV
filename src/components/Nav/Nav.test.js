import React from 'react';
import Nav from './Nav';
import { shallow } from 'enzyme'

describe('Nav', () => {


  it('should match a snapshot as expected', () => {
    const wrapper = shallow(<Nav />)

    expect(wrapper).toMatchSnapshot()
  })
})
