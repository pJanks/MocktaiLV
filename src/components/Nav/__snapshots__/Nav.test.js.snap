// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should match a snapshot as expected 1`] = `
<nav>
  <button>
    <Link
      className="link"
      to="/"
    >
      Homepage
    </Link>
  </button>
  <button>
    <Link
      className="link"
      to="/OffTheWagon"
    >
      Off The Wagon
    </Link>
  </button>
  <button>
    <Link
      className="link"
      to="/Favorites"
    >
      Favorites
    </Link>
  </button>
</nav>
`;
